OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 6

# Description
Lint fix for Reports component, this is phase 6.

## Files changed
All js/jsx file in /src/components/Reports/TasksTable

eslint test:
<img width="1490" alt="CleanShot 2023-09-30 at 18 19 16@2x" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/facc863b-c474-458b-b7c6-516cfc532d35">



# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- ` npx eslint "src/components/Reports/TasksTable/*.js*" `

- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!
